libgadget/exchange.c

```diff
@@ -44,10 +43,10 @@ typedef struct {
  * exchange particles according to layoutfunc.
  * layoutfunc gives the target task of particle p.
 */
-static int domain_exchange_once(int (*layoutfunc)(int p), ExchangePlan * plan, int do_gc);
-static void domain_build_plan(int (*layoutfunc)(int p), ExchangePlan * plan);
+static int domain_exchange_once(ExchangePlan * plan, int do_gc);
```

Comment:
Shall we lift the function point type with a typedef? It feels like the right time to do it now; e.g. ExchangeLayoutFunc?

Reply:
Done!